.gitignore

```diff
@@ -130,4 +130,7 @@ env/
 
 # Crash log files
 crash.log
-crash.*.log
\ No newline at end of file
+crash.*.log
+
+# minikube environment secrets file
+env-secrets.yaml
```

Comment:
Could be worth adding this to the `.dockerignore` as well to make sure that your images don't accidentally get this file built in too (fine if you always build them in CI, but if anyone does it manually it could be a risk)